### PR TITLE
added tokenResult, tokensResponse

### PR DESCRIPTION
**Pull request by anjbur**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [ ] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

A few sentences describing the overall goals of the pull request.

## Related PRs or issues

List related PRs and other issues.

## Misc

Add what's missing, notes on what you tested, additional thoughts or questions.

### CODE REVIEW

File: AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSTokenResult.m [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#import <Foundation/Foundation.h>
```
Comment (jaelim-ms) [inline AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSTokenResult.m]: Probably not necessary. It is already imported in header file.

File: AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSTokensResponse.h [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#import <Foundation/Foundation.h>
```
Comment (jaelim-ms) [inline AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSTokensResponse.h]: Probably not necessary. It is already imported in header fil

File: AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSTokensResponse.m [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#import <Foundation/Foundation.h>
```
Comment (jaelim-ms) [inline AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSTokensResponse.m]: Probably not necessary. It is already imported in header fil

File: AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSTokenResult.m (hunk 1 above)
Comment (anjbur) [reply AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSTokenResult.m]: Got it, removed.

File: AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSTokensResponse.h (hunk 2 above)
Comment (anjbur) [reply AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSTokensResponse.h]: Got it, removed.

File: AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSTokensResponse.m (hunk 3 above)
Comment (anjbur) [reply AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSTokensResponse.m]: Got it, removed.

File: AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSTokenResult.m [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+@synthesize partition = _partition;
+@synthesize dbAccount = _dbAccount;
+@synthesize dbName = _dbName;
+@synthesize dbCollectionName = _dbCollectionName;
+@synthesize token = _token;
+@synthesize status = _status;
+
+- (instancetype)initWithPartition:(NSString *)partition
+                     andDbAccount:(NSString *)dbAccount
```
Comment (MehrdadMzfr) [inline AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSTokenResult.m]: NIT: remove and prefix e.g. 
dbAccount: :(NSString *)dbAccount
dbName::(NSString *) dbName
...

**Comment by MehrdadMzfr** [comment]
since you added files, you should also check in the project.pbxproj file

File: AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSTokenResult.m (hunk 4 above)
Comment (anjbur) [reply AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSTokenResult.m]: made the change 👍

**Pull request merged**